pages/transactions/[id].js

```diff
@@ -73,6 +75,14 @@ export default function TransactionDetailsPage({
 
   return (
     <main>
```

Comment:
You already have a main on this site like you can see in the dev tools. This one would be another main. That should not happen. Make sure each page has only one main element.